files/messages/welcome.txt

```diff
@@ -1,9 +1,10 @@
 Hello everyone, and thank you for signing up for our upcoming instructor training session. In order to prepare:
 
-1.  Please read these two short papers, which provide a brief overview of some key evidence-based results in teaching:
+1.  Please read these three short papers, which provide a brief overview of some key evidence-based results in teaching:
 
     * "The Science of Learning" (https://swcarpentry.github.io/instructor-training/files/papers/science-of-learning-2015.pdf)
     * "Success in Introductory Programming: What Works?" (https://swcarpentry.github.io/instructor-training/files/papers/porter-what-works-2013.pdf)
+    * "What Can I Do Today to Create a More Inclusive Community in CS?" (https://swcarpentry.github.io/instructor-training/files/papers/lee-create-inclusive-community-2015.pdf)
```

Comment:
This does not really "provide a brief overview of some key evidence-based results in teaching"...